contracts/Core/StakeManager.sol

```diff
@@ -134,6 +134,8 @@ contract StakeManager is Initializable, ACL, StakeStorage, Pause {
 
     event Delegated(uint256 epoch, uint256 indexed stakerId, address delegator, uint256 previousStake, uint256 newStake, uint256 timestamp);
 
+    event DelegationEnabled(uint256 indexed stakerId, address staker, bool delegationEnabled);
```

Comment:
Given event is also going to be fired when delegation is disabled 
so maybe "DelegationEnabled" is not correct as the name 
can we have DelegationAcceptanceChanged ?